semantics.json

```diff
@@ -20,6 +20,14 @@
     "importance": "high",
     "description": "Learners must provide an answer in order to proceed"
   },
+  {
+    "label": "Characters Limit",
+    "name": "charactersLimit",
+    "type": "number",
+    "importance": "high",
+    "default": "200",
```

Comment:
Yes, I think it is needed because we allow the user to change this limit from the editor, and if the user decided to submit an empty field, that will break the calculation logic. I am still open to suggestions!

Reply:
Not sure if I understand this correctly: I was under the assumption that the character limit was optional. If it is, then a default value should be set when instantiating the content type anyway. All non-optional values need to be sanitized if they are required. One cannot assume that all parameters are set. For instance, when subcontent is used in Course Presentation, the visual editor will instantiate subcontent with empty parameters - this must not crash content.

Reply:
Guess we'll have to wait for some feedback from @janlindso or Knut-Inge then. I don't know if a limit should be mandatory. If it is, then yes, importance should be "high".

Reply:
It's been updated and revised.